AmplifyPlugins/Auth/AWSAuthPlugin/Support/Utils/AWSCredentials+Extension.swift

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import AWSCore
+
+extension AWSCredentials {
+
+    func toAmplifyAWSCredentials() -> AuthAWSCognitoCredentials {
+        // Credentials are fetched through Cognito Identity Pool and thus these are temporary credentials
```

Comment:
Are we sure about this? What's to prevent someone setting up UserPools with no IdentityPool, and then calling `toAmplifyAWSCredentials()`?